dockerfiles/Dockerfile.centos7

```diff
@@ -12,18 +12,27 @@ RUN sed -i -e "s/^mirrorlist=http:\/\/mirrorlist.centos.org/#mirrorlist=http:\/\
     cyrus-sasl-lib cyrus-sasl-devel openssl openss-libs openssl-devel \
     postgresql-libs postgresql-devel postgresql-server postgresql libyaml-devel && \
     yum install -y epel-release && \
-    yum install -y cmake3
+    yum install -y cmake3 && \
+    yum install -y centos-release-scl && \
+    sed -i -e "s/^mirrorlist=http:\/\/mirrorlist.centos.org/#mirrorlist=http:\/\/mirrorlist.centos.org/g" /etc/yum.repos.d/CentOS-SCLo-scl.repo && \
+    sed -i -e "s/^# baseurl=http:\/\/mirror.centos.org/baseurl=http:\/\/vault.centos.org/g" /etc/yum.repos.d/CentOS-SCLo-scl.repo && \
+    sed -i -e "s/^mirrorlist=http:\/\/mirrorlist.centos.org/#mirrorlist=http:\/\/mirrorlist.centos.org/g" /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo && \
+    sed -i -e "s/^#baseurl=http:\/\/mirror.centos.org/baseurl=http:\/\/vault.centos.org/g" /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo && \
+    yum install -y devtoolset-8
 
 COPY . /src/
 WORKDIR /src/build
 
-RUN cmake3 -DCMAKE_INSTALL_PREFIX=/opt/fluent-bit/ -DCMAKE_INSTALL_SYSCONFDIR=/etc/ \
+# /opt/rh/devtoolset-8/enable is from centos-release-scl package.
+# hadolint ignore=SC1091
+RUN source /opt/rh/devtoolset-8/enable && \
```

Comment:
Ah, this is the magic to support it on vanilla - are we good with licensing for it?

Reply:
We could also just disable for the older targets